TIS-100_Commented/10981.0.txt

```diff
@@ -0,0 +1,30 @@
+@0
+
+
+@1
+MOV UP, ACC # grab from above and store
+ADD ACC # add ACC to ACC
```

Comment:
Comments should focus on "why" rather than "what" -- e.g. here, we want to double the value in `ACC`